### ISSUE TITLE
error

### ISSUE THREAD

**Issue by eternal8080** [yelboudouri/EmoNeXt#3]
when run train.py, in scheduler.py, error: cannot import name 'LRScheduler' from 'torch.optim.lr_scheduler'
